### PR TITLE
#31762 Fake data of user, location, yard, cost-time, booking

### PR DESCRIPTION
**Pull request by lunguyendev**

### CODE REVIEW

**Comment by lunguyendev** [comment]
Screenshot from 2020-11-12 14-45-55

**Review by viethuyunh020998 — APPROVED** [review]

**Review by nguyentienanhvuongchuse — APPROVED** [review]
oke

**Comment by lunguyendev** [comment]
ready

File: app/models/booking.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Booking < ApplicationRecord
   belongs_to :time_cost
   belongs_to :user
+  enum status: {pending: 0, accept: 1, cancel: 2}
```
Comment (vanvtt-0952) [inline app/models/booking.rb]: reject nữa nhé :p 
cancel là phía ng dùng, reject là phía admin

**Comment by lunguyendev** [comment]
ready

**Pull request merged**